pipelines/crawler/cgu/utils.py

```
@@ -145,17 +146,26 @@ def download_file(
 
-        status = requests.get(url).status_code == 200
+        if dataset_id == "br_cgu_beneficios_cidadao":
+            url = url.rstrip("/")
+
+            headers = {
+                "User-Agent": constants.BROWSERS_USER_AGENT.value["chrome"]
+            }
+            status = source_url_is_available(url=url)
+        else:
+            headers = None
+            status = requests.get(url=url, headers=headers).status_code == 200
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)